### PR TITLE
Added Du-Form (#45)

### LINKED ISSUE #45: German translations: "Du-Form"
Change the german translations to consistently use the "Du-Form" instead of the "Sie-Form" (https://github.com/orgs/Bootstrap-Academy/discussions/39#discussioncomment-7219595).

### PR DESCRIPTION
**Pull request by josiebytes**
## Description
Removed the formal form and added the personal form as mentioned in issue #45. I deleted my old pr on accident but this one does have all the changes.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Translation updates (fix/improve or add new translations)

Fixes Bootstrap-Academy/Bootstrap-Academy/issues/45

My Bootstrap Academy username: Joshiy13

### CODE REVIEW

**Review by SpotlightForBugs — APPROVED** [review]
Sehr gut, endlich einheitlicher

File: locales/de.json [hunk 1]
```diff
@@ -30,45 +30,45 @@
     "Excel": "Excel",
     "Python": "Python",
     "PremiumEndsIn": "Ihr Premium-Abonnement endet",
```
Comment (Defelo) [inline locales/de.json]:
```suggestion
    "PremiumEndsIn": "Dein Premium-Abonnement endet",
```

File: locales/de.json [hunk 2]
```diff
@@ -30,45 +30,45 @@
...
-    "ButAdditionalSubscription": "Wenn Sie ein zusätzliches Abonnement erwerben, wird die Laufzeit Ihres bestehenden Abonnements verlängert.",
+    "ButAdditionalSubscription": "Wenn Du ein zusätzliches Abonnement erwirbst, wird die Laufzeit deines bestehenden Abonnements verlängert.",
     "More": "mehr",
     "NoMoreUser": "Keine Benutzer mehr",
     "PowerPoint": "PowerPoint",
     "Quizzes": "Quizze",
-    "ClickForExampleCode": "Klicken Sie auf „Für Beispielcode“.",
+    "ClickForExampleCode": "Klicke auf „Für Beispielcode“.",
     "CompilationError": "Kompilierungsfehler ❌",
     "NoOutput": "Es gibt keine Ausgabe für Ihren bereitgestellten Code🫣",
```
Comment (Defelo) [inline locales/de.json]:
```suggestion
    "NoOutput": "Es gibt keine Ausgabe für deinen Code 🫣",
```

File: locales/de.json [hunk 3]
```diff
@@ -30,45 +30,45 @@
...
     "SubmissionSolved": "Gelöst✅",
     "PendingResult": "Warte! Wir verarbeiten Ihren angegebenen Code",
```
Comment (Defelo) [inline locales/de.json]:
```suggestion
    "PendingResult": "Warte! Wir verarbeiten deinen Code",
```

**Review by Defelo — CHANGES_REQUESTED** [review]

**Review by Defelo — APPROVED** [review]

**Pull request merged**